src/util/util.ts

```diff
@@ -0,0 +1,11 @@
+export function toBoolean(value: any): boolean {
+  return value === '' ? true : !!value;
```

Comment:
Ha! This is _exactly_ the discussion I wanted to trigger with this PR :-) Also why `"false"` doesn't get converted to `false`? We might rename this function to better indicate that this is for inputs, but the most important part for me is to agree on the semantic